src/Compilers/CSharp/Test/Semantic/FlowAnalysis/FlowDiagnosticTests.cs

```diff
@@ -1344,7 +1344,10 @@ static void Main(string[] args)
     Diagnostic(ErrorCode.ERR_UseDefViolationProperty, "x2").WithArguments("x2").WithLocation(17, 34),
     // (14,12): error CS0843: Backing field for automatically implemented property 'Program.x' must be fully assigned before control is returned to the caller. Consider calling the default constructor from a constructor initializer.
     //     public Program()
-    Diagnostic(ErrorCode.ERR_UnassignedThisAutoProperty, "Program").WithArguments("Program.x").WithLocation(14, 12)
+    Diagnostic(ErrorCode.ERR_UnassignedThisAutoProperty, "Program").WithArguments("Program.x").WithLocation(14, 12),
+    // (14,12): error CS0843: Backing field for automatically implemented property 'Program.x2' must be fully assigned before control is returned to the caller. Consider calling the default constructor from a constructor initializer.
```

Comment:
>error CS0843 [](start = 16, length = 12)

Adding a compiler error breaks compatibility, so that is likely not the right approach.

Reply:
yeah I never expected this approach to get accepted as a "quick" fix when it adds compiler errors. I mainly included it to learn from the discussion about it but looking back at it now, it feels like a "dumb" decision and mostly waste of time. I'll remove it tonight so we can move on with the fix 😄